final_prep/README.md

```diff
@@ -23,52 +23,64 @@ Some things to consider as you refactor include...
+- [ X] Study/Project work time (usually about 25 hours/week outside of class time)
+- [ X] Personal, Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+
+- [Week 1](https://user-images.githubusercontent.com/102967531/164868073-e0358a99-acd6-4637-a935-7bfcb59227bb.png)
+
+- [Week 2](https://user-images.githubusercontent.com/102967531/164868032-d13e6e2e-ec80-4c95-8ec8-f7b1c46adb8b.png)
+
+- [Week 3](https://user-images.githubusercontent.com/102967531/164868145-c9d6c9bb-9229-411a-bcc1-0ecf81e11c9d.png)
+
 
 ## Mentorship Prep
 
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
 
-- [ ] [Create a profile on Turing Community Connect by following these directions](https://docs.google.com/document/d/1vpyKGu92l1HGkJzULNcyyE72946f4QO1DhQgIz3v1E0/edit?usp=sharing)
-- [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+
+- [ X] [Create a profile on Turing Community Connect by following these directions](https://docs.google.com/document/d/1vpyKGu92l1HGkJzULNcyyE72946f4QO1DhQgIz3v1E0/edit?usp=sharing)
+- [ X] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
+
+  - [ X] Add link to your gist here: [Mentee Gist](https://gist.github.com/bensjsilverstein/dad5f2db88549fab56f242a01a468b3c)
+
 
 ## Lesson Prep
 
 You've learned a lot about how to take strong notes during Mod 0. Show us your skills while you learn how to pre-teach content for your first lesson in Mod 1!
 
-- [ ] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
-  - [ ] Add a link to your gist here:
+- [ X] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
+  - [ X] Add a link to your gist here: [Lesson Prep Gist](https://gist.github.com/bensjsilverstein/487bc109e170c6ff3c7bd0e239ebd0d2)
 
 ## Group Work Prep
 
 As part of Turing's project-based learning approach, you will often be working in pairs or larger groups. In order to set yourself (and your team) up for success, it is important to ensure you are prepared to be an equitable contributor and teammate.
 
-- [ ] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
-  - [ ] Add a link to your gist here: 
+- [ X] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
+
+  - [ X] Add a link to your gist here: [DTR Guiding Questions Gist](https://gist.github.com/bensjsilverstein/b43ac7c4c11a497bd4b26a42d3b6d7eb)
```

Comment:
This looks great. I hope your father is doing ok and on his way to recovery! It's great that it's on your radar you may have to take trips, and to share with group mates so they understand you're dealing with that in the personal background and may need some time to focus on that, travel, etc.